page/ts/index.ts

```diff
@@ -60,5 +60,17 @@ $(document).ready(function(){
     $('.team .see-more').click(function(){        
         $(this).text().trim().toLowerCase() == seeMore.trim().toLowerCase() ? $(this).text(seeLess) : $(this).text(seeMore);
         $('.team .is-hidden').fadeToggle("slow","linear");        
-    });
+    });    
+});
+
+$(window).scroll(function(e){
```

Comment:
great! could you also change color of that border to #f0f0f1? There is a variable called `$navbar-default-border`